cellect_env.rb

```diff
@@ -47,19 +59,32 @@ def load_db_yaml
       @pg_host = ENV['PG_PORT_5432_TCP_ADDR']
       @pg_port = ENV['PG_PORT_5432_TCP_PORT']
       @pg_db = ENV['PG_ENV_DB']
-      @pg_user = ENV['PG_ENV_PG_USER']
-      @pg_pass = ENV['PG_ENV_PASS']
       @pg_pool = ENV['PG_ENV_POOL']
+      @pg_user = ENV['PG_ENV_POSTGRES_USER']
+      @pg_pass = ENV['PG_ENV_POSTGRES_PASSWORD']
     end
   end
 
-  def load_zk_yaml
-    begin
-      zookeepers = YAML.load(File.read("/production_config/zookeeper.yml"))
-      zk = zookeepers[@environment]
-      @zk_url = zk['url']
-    rescue Errno::ENOENT
-      @zk_url = "#{ENV["ZK_PORT_2181_TCP_ADDR"]}:#{ENV["ZK_PORT_2181_TCP_PORT"]}"
+  def default_conn_pool_size
+    @default_conn_pool_size ||= 8
+  end
+
+  def connection_pool_value
```

Comment:
I think this is loading before the puma fork.  Trying to benchmark this, it's running out of available connections.

Chances are, this needs to be `PUMA_THREADS * PG_POOL_SIZE`